upup/pkg/fi/cloudup/iam_builder.go

```diff
@@ -105,6 +105,38 @@ func (b *IAMPolicyBuilder) BuildAWSIAMPolicy() (*IAMPolicy, error) {
 			Action:   []string{"elasticloadbalancing:*"},
 			Resource: []string{"*"},
 		})
+
+		// Restrict the KMS permissions to only the keys that are being used
+		var set = make(map[string]bool)
```

Comment:
Thanks for the pointers, will keep in mind for the future! I like the look of the updated code in https://github.com/kubernetes/kops/pull/459.